demo/templates/demo/analytics.html

```diff
@@ -0,0 +1,2 @@
+<script async defer data-hostname="demo.mathesar.org" src="https://scripts.simpleanalyticscdn.com/latest.dev.js"></script>
```

Comment:
Please use our custom subdomain for the analytics scripts, see https://github.com/centerofci/mathesar-website/blob/master/_layouts/default.html#L52